### ISSUE TITLE
Support for newer PMS versions

### ISSUE THREAD

**Issue by Just-Insane** [munnerz/kube-plex#53]
Are there any plans to support newer Plex versions?

The imaged used in values.yaml is a year old at this point, and trying to use anything newer fails to start a transcoder pod.

The error is "Conversion failed. A required codec could not be found or failed to install."

It also appears kube-plex's docker image has not been updated in a year either.

**Comment by Just-Insane** [comment]
It is possible it was just a weird issue with the media.

Tried it again with the latest tag and it seemed to be working.

Off topic, but what’s the best way to get metrics about the transcode?

**Comment by ivveh** [comment]
I'm currently experimenting with kube-plex and so far I have found failures that are related to EAE codecs. However, if the media does not require those codecs, it spawns pods and transcodes.
Anyways, the ENV variable in the PMS is pointed into /tmp, i.e: EAE_ROOT: /tmp/pms-71e1831f-bbd9-46a8-badf-1275bccdeb0e/EasyAudioEncoder
But the containers /tmp is empty...

**Comment by ivveh** [comment]
Forgot to add, altho this problem exists, new version run just fine.

**Comment by Just-Insane** [comment]
As I recall, the errors reported when trying to run a newer version were all codec errors, so this would make lots of sense.

Do you happen to know where to get the codecs from?


________________________________

**Comment by ivveh** [comment]
Nope, I just updated the helm chart with the latest tag of the PMS and then just upgraded it. I can see the /tmp/<randomuuid>/EasyAudioEncoder/ dir in ENV on the elastic pods as well as it being empty when I look inside them.. I have no plugins or codecs installed. I'm guessing the dir can be pointed to /transcode in order to get it to work but the plex pod would probably need it too, I guess. I don't really know much about plex, just a simple user :)

**Comment by ivveh** [comment]
Or it simply doesn't have correct permissions to /tmp.. which is usually a problem with containers.
